scripts/source-maintenance.sh

```diff
@@ -35,6 +35,14 @@ while [ $# -ge 1 ]; do
         KeepGoingDirective=$1
         shift
         ;;
+    --skip-astyle-version-check)
+        SkipAstyleVersionCheck=yes
+        shift
+        ;;
+    --verbose|-v)
```

Comment:
I do not think we should add this option (in this PR) because bad verbose output (similar to what is being proposed here) can already be (presumably) accomplished with 'sh -x` while high-quality verbose output would require more work (and a dedicated PR).

Reply:
sh -x is very verbose and can be confusing; it's meant to debug the script, not follow it along

Reply:
Your observations are correct but do not address my change request. If you want to add proper verbose logging for "following the script", I cannot stop you (even if I personaly consider that feature unnecessary), but the current PR does not do it.

Reply:
Resolving thread as this option no longer added at all.